WordPress/src/test/java/org/wordpress/android/ui/posts/prepublishing/visibility/usecases/UpdatePostStatusUseCaseTest.kt

```diff
@@ -77,6 +82,18 @@ class UpdatePostStatusUseCaseTest : BaseUnitTest() {
         assertThat(editPostRepository.getPost()?.status).isEqualTo(expectedPostStatus.toString())
     }
 
+    @Test
+    fun `verify that when updatePostStatus is called with PRIVATE Visibility dateTimeUtilsWrapper is used`() {
```

Comment:
We might want to consider replacing this test with 
`verify that when updatePostStatus is called with PRIVATE Visibility the creation date is set to now`. Mocking the `currentTimeInIso8601` and verifying it's set to the mocked value will also verify that the method was called.

Reply:
Exactly. Thanks. I woke up with this very thought 😅

Reply:
Resolved this by creating tests for the new use case 937f9e6